src/components/ComicBook/MathQuestions.tsx

```diff
@@ -0,0 +1,142 @@
+import { Page, Question } from "../../types";
+import { useEffect, useState } from "react";
+import { SyntheticEvent } from "react";
+import comicService from "../../services/comicService";
+import axios from "axios";
+const MathQuestions = ({
+  stateKey,
+  setKey,
+  comicName,
+  comic,
+  page,
+}: {
+  stateKey: string | undefined;
+  setKey: React.Dispatch<React.SetStateAction<string | undefined>>;
+  comicName: string;
+  comic: Page[];
```

Comment:
Are you sure you need to send comic and page number, and then index page from comic? Is it possible to give one Page as a prop. Page should have questions in it?